src/libs/OptionsListUtils.ts

```diff
@@ -2352,12 +2355,13 @@ function getFirstKeyForList(data?: Option[] | null) {
     return firstNonEmptyDataObj?.keyForList ? firstNonEmptyDataObj?.keyForList : '';
 }
 
-function getPersonalDetailSearchTerms(item: Partial<OptionData>) {
-    return [item.participantsList?.[0]?.displayName ?? '', item.login ?? '', item.login?.replace(CONST.EMAIL_SEARCH_REGEX, '') ?? ''];
+function getPersonalDetailSearchTerms(item: Partial<OptionData>, shouldIncludeCurrentUser = false) {
```

Comment:
Why are we adding `shouldIncludeCurrentUser` param here? Shouldn't we use `getCurrentUserSearchTerms` for the "current user" option